content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -52,11 +65,15 @@ If there are other branches you want to exclude, add them on the Test Service Se
 
```

Comment:
Does the #limitations link need to be updated now that the header has changed to "Out-of-the-box configuration limitations"